packages/viz/src/BarChart/stories/BarChart.stories.tsx

```diff
@@ -494,3 +497,110 @@ export const CustomEchartsOptions: StoryObj<HvBarChartProps> = {
   },
   render: () => <CustomEchartsOptionsStory />,
 };
+
+export const WithFiltering: StoryObj<HvBarChartProps> = {
+  parameters: {
+    docs: {
+      description: {
+        story:
+          "This sample illustrates how to filter the data using the `filters` prop.",
+      },
+    },
+  },
+  render: () => {
+    const [sales, setSales] = useState([0, 7000]);
+    const [country, setCountry] = useState("");
+    const [filters, setFilters] = useState<HvChartFilter[]>([]);
+
+    const data = [
+      {
+        label: "Portugal",
+        value: 3600,
+      },
+      {
+        label: "USA",
+        value: 1500,
+      },
+      {
+        label: "India",
+        value: 6700,
+      },
+      {
+        label: "China",
+        value: 4500,
+      },
+      {
+        label: "France",
+        value: 3200,
+      },
+      {
+        label: "UK",
+        value: 2700,
+      },
+      {
+        label: "Japan",
+        value: 5800,
+      },
+    ];
+
+    useEffect(() => {
+      setFilters([
+        {
+          field: "Country",
+          operation: "contains",
+          value: country,
+        },
+        {
+          field: "Sales",
+          operation: "between",
+          value: [sales[0], sales[1]],
+        },
+      ]);
+    }, [country, sales]);
+
+    const handleSliderChange = (values) => {
+      setSales(values);
+    };
+
+    const handleCountryChange = (event, value) => {
+      setCountry(value);
+    };
+
+    return (
+      <>
+        <div
+          style={{
+            width: "100%",
+            display: "flex",
+            justifyContent: "center",
+            gap: 40,
+          }}
+        >
+          <HvSlider
+            minPointValue={0}
+            maxPointValue={7000}
+            values={sales}
+            onChange={handleSliderChange}
+            hideInput
+            label="Sales"
+            style={{ width: 400 }}
+          />
+          <HvInput
```

Comment:
The input width changes when we type something 👀 Can we set the width? Or maybe a dropdown (`HvSelect`) with multiple selection would be better for the sample: if I type "india" instead of "India" nothing shows because of the capitalised letter.